reference/outlook/preview/Recurrence.md

```diff
@@ -0,0 +1,201 @@
+
+
+# Recurrence
+
+The `recurrence` object provides methods to get and set the recurrence pattern of appointments but only to get the recurrence pattern of meeting requests. It will have a dictionary with the following keys: `seriesTime`, `recurrenceType`, `recurrenceProperties`, and `recurrenceTimeZone` (optional).
```

Comment:
appointments and meetings

Reply:
My understanding is get/set appointments but get meetings?

Reply:
get and set meetings/appointments but only get meeting requests (email in inbox of user that was invited)